src/utils.js

```diff
@@ -665,3 +666,29 @@ export function useSessionStorage(key, initialValue) {
 
     return [storedValue, setValue];
 }
+
+export async function getGoogleClassroomCourses(
```

Comment:
Write documentation for this - what are the inputs and output of this function? give example/description of how this util should be used. This is a google blog post on how this should be done: https://gomakethings.com/whats-the-best-way-to-document-javascript/